adafruit_azureiot/iot_error.py

```diff
@@ -1,24 +1,7 @@
-# The MIT License (MIT)
+# SPDX-FileCopyrightText: 2020 Jim Bennet for Adafruit Industries
```

Comment:
additional author `Elena Horton` listed below. There are some more of these as well, I wont comment all of them but I can come back tomorrow night and push fixes to add this name on the copyright line.